### ISSUE TITLE
doesn't seem to work any more

### ISSUE THREAD

**Issue by hojnrq** [Cih2001/KodiSubsceneAddon#4]
Has this been abandoned?

It's been a few months since this addon came up with any results for any of the movies I tried within Kodi 18. It just stopped getting hits - used to work. 

However, If I search on the subscene.com website, there are always subtitles available for the languages I specified in Kodi.

**Comment by Cih2001** [comment]
Hey, No it is not abandoned. Can you please send me more info? what movies for example are you not able to get the subtitle for? does it fail with all subtitles or only a particular language? can you paste the kodi log here?

**Comment by viktor-vintertass** [comment]
Tried it today with a movie that I found using my computer to search.
The addon did not show any results.
So not working.

What kind of logs do you need?

**Comment by Cih2001** [comment]
Hi @viktor-morin, I have noticed that it sometimes does not work for some movies. One of the issues I found is that the production year on the Subscene website is different than the production year of that movie in your library. This addon relies on the consistency of such data. I have to fix that.

Anyway, for now, I need this info to reproduce the issue:

1. Name of the movie (the one shown in your library)
2. Production Year (the one shown in your library)
3. File name. (The file name you have stored the movie under its name)
